packages/contracts/contracts/DependencyRegistryV0.sol

```diff
@@ -114,39 +135,58 @@ contract DependencyRegistryV0 is
         _transferOwnership(_adminACLContract);
     }
 
+    /**
+     * @notice Adds a new license type that can be used by dependencies.
+     */
+    function addLicenseType(bytes32 _licenseType) external {
+        _onlyAdminACL(this.addLicenseType.selector);
+        // @dev the add function returns false if set already contains value
+        require(_licenseTypes.add(_licenseType), "License type already exists");
+        require(
+            _licenseType != bytes32(""),
+            "License type cannot be empty string"
+        );
```

Comment:
nit: swap these two statements, as that order more closely follows the [checks-effects-interactions pattern](https://fravoll.github.io/solidity-patterns/checks_effects_interactions.html)